TradeItIosTicketSDK2/TradeItOrdersViewController.swift

```diff
@@ -86,8 +88,6 @@ class TradeItOrdersViewController: TradeItViewController, TradeItOrdersTableDele
                 withLinkedBroker: self.linkedBrokerAccount?.linkedBroker,
                 onViewController: self
             )
```

Comment:
There is an issue with dummy 600 though. We will get an alert with session expired - cancel - refresh. With the refresh, the screen won't be updated.
If we add the refresh in an onFinished, this will loop in case of error:
```
self.alertManager.showAlertWithAction(
                error: error,
                withLinkedBroker: self.linkedBrokerAccount?.linkedBroker,
                onViewController: self,
                onFinished: {
                    self.refreshRequested(onRefreshComplete: onRefreshComplete)
                }
            )
```
Thoughts ?

Reply:
Maybe we should call showAlertWithAction only is it is a 700 error ?

Reply:
Is `onFinished` called if a user clicks Cancel?

Reply:
@jsom Yes it is